install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,674 @@
+|APBs that match a regex in blacklist.
+
+|None
+|No APBs from the registry.
+|All APBs from that registry.
+|===
+
+For example:
+
+.Whitelist Only
+[source,yaml]
+----
+white_list:
+  - "foo.*-apb$"
+  - "^my-apb$"
+----
+
+Anything matching on `foo.*-apb$` and only `my-apb` will
+be allowed through in this case. All other APBs will be rejected.
+
+.Blacklist Only
+[source,yaml]
+----
+black_list:
+  - "bar.*-apb$"
+  - "^foobar-apb$"
+----
+
+Anything matching on `bar.*-apb$` and only `foobar-apb` will be blocked in this
+case. All other APBs will be allowed through.
+
+.Whitelist and Blacklist
+[source,yaml]
+----
+white_list:
+  - "foo.*-apb$"
+  - "^my-apb$"
+black_list:
+  - "^foo-rootkit-apb$"
+----
+
+Here, `foo-rootkit-apb` is specifically blocked by the blacklist despite its
+match in the whitelist because the whitelist match is overridden.
+
+Otherwise, only those matching on `foo.*-apb$` and `my-apb` will be allowed
+through.
+
+.Example Broker Configuration `registry` Section:
+[source,yaml]
+----
+registry:
+  - type: dockerhub
+    name: dockerhub
+    url: https://registry.hub.docker.com
+    user: <user>
+    pass: <password>
+    org: <org>
+    white_list:
+      - "foo.*-apb$"
+      - "^my-apb$"
+    black_list:
+      - "bar.*-apb$"
+      - "^foobar-apb$"
+----
+
+[[oab-broker-config-registry-local]]
+=== Local OpenShift Container Registry
+
+Using the `local_openshift` type will allow you to load APBs from the
+OpenShift Container Registry that is internal to the {product-title} cluster.
+You can configure the namespaces in which you want to look for published APBs.
+
+[source,yaml]
+----
+registry:
+  - type: local_openshift
+    name: lo
+    namespaces:
+      - openshift
+    white_list:
+      - ".*-apb$"
+----
+
+[[oab-broker-config-registry-rhcc]]
+=== Red Hat Container Catalog Registry
+
+Using the `rhcc` type will allow you to load APBs that are published to the
+link:https://access.redhat.com/containers[Red Hat Container Catalog] (RHCC)
+registry.
+
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: https:://registry.access.redhat.com
+    white_list:
+      - ".*-apb$"
+----
+
+[[oab-broker-config-isv]]
+=== ISV Registry
+
+Using the `openshift` type allows you to load APBs that are published to the
+ISV container registry at
+link:https://registry.connect.redhat.com[registry.connect.redhat.com].
+
+[source,yaml]
+----
+registry:
+  - name: openshift
+    type: openshift
+    auth_type: config <1>
+    user: <user>
+    pass: <password>
+    url: https://registry.connect.redhat.com
+    images: <2>
+      - <image_1>
+      - <image_2>
+    white_list:
+      - ".*-apb$"
+----
+<1> Using the `openshift` registry type currently requires that `auth_type` be
+declared in the configuration, either to `config` or `secret`, due to a bug that
+will be addressed in a future release
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1526949[BZ#1526949]). The
```

Comment:
Actually, for 3.7 version, it works fail only if specify the `auth_type: config`. But, for 3.9 version, it is not necessary. 
I think we should clarify this point and add the description of the `file` type.